ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/KsqlStructuredDataOutputNode.java

```diff
@@ -83,15 +80,8 @@ public SourceName getIntoSourceName() {
   }
 
   @Override
-  public QueryId getQueryId(final QueryIdGenerator queryIdGenerator) {
-    final String base = queryIdGenerator.getNext().toUpperCase();
```

Comment:
What happened to this `toUpperCase()` call? I don't see it in the migrated code.

Reply:
they all actually return numbers, so it's kind-of irrelevant. I'll add it back for consistency, but we should probably change the API to just return a number.

Reply:
Looks like it was added back in one place but not another (https://github.com/confluentinc/ksql/blob/master/ksqldb-engine/src/main/java/io/confluent/ksql/engine/QueryIdUtil.java#L58) :) If they're always numbers I'm fine with leaving it off, though.

Reply:
🤦 shouldn't make a difference, but i'll change it in the next PR to be safe